lms/services/lti_role_service.py

```diff
@@ -0,0 +1,35 @@
+from typing import List
+
+from sqlalchemy.orm import Session
+
+from lms.models import LTIRole
+
+
+class LTIRoleService:
+    """A service for dealing with LTIRole objects."""
+
+    def __init__(self, db_session: Session):
+        self._db = db_session
+
+    def get_roles(self, role_strings: List[str]) -> List[LTIRole]:
+        """Get a list of role objects for the provided strings."""
```

Comment:
This could accept an LTIUser as well, but it seems overly coupled to me. It feels right in the assignment service though.